tests/Microsoft.Identity.Test.Integration.netfx/Infrastructure/IgnoreOnOneBranch.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Identity.Test.Integration.Infrastructure
+{
+    internal class IgnoreOnOneBranchAttribute : TestMethodAttribute
```

Comment:
can this be made generic so we can add more ignore cases in the future? not just onebranch?

Reply:
Yeah, Ill do it on another PR. Trying to get this one in before we run into more build issues